fluent-dom/test/index.js

```diff
@@ -13,3 +13,6 @@ Object.keys(document.defaultView).forEach(property => {
 global.navigator = {
   userAgent: 'node.js'
 };
+
+// silence console logging during tests
+console.warn = () => {}
```

Comment:
Suggestion: I don't think this is a good change. It would be better to add suppressions per-test. This is a global change for every test, and it could be very surprising as to why this function doesn't work. I generally treat console as a public API, and one that should be handled explicitly, and asserted in tests.

Reply:
Ok, will change. I'll bump up `sinon` to the root package.json and use that in these tests as well.